package.json

```diff
@@ -27,6 +27,10 @@
       "git add"
     ]
   },
+  "browserslist": [
+    "Chrome 50",
+    "Firefox 50"
```

Comment:
Ideally, this would say “`last 2 versions`”, but that would make cloning this repo + `npm install` + then running the benchmark non-deterministic. Maybe we should start checking in `package-lock.json`?

Reply:
I don't have a strong opinion on this one. I avoided checking in `package-lock.json` as it was always modified essentially, even when it shouldn't (logically). I just didn't spend the time figuring out how this works.

Reply:
I'm not sure if the `package-lock.json` allows to save this kind of information. However, I think is a good idea to commit the lock file to the repo. Maybe in other PR?

Reply:
I meant that `package-lock.json` would lock all dependencies, including `browserslist`. Thus, given some input CSS and an browser list expression like `"last 2 versions"`, that would always map to the same list of browser versions.